docs/TuxOwnerClass.md

```diff
@@ -0,0 +1,269 @@
+# Tux/React.OwnerClass
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Owner Class is an opinionated [React](http://facebook.github.io/react/) class designed to manage state in an application.  It is also an excellent class to use as an endpoint for a client side route in the app.  The OwnerClass possesses methods that allow it to more easily connect with a store or stores and expose its properties to child components without having to cascade those properties down as per the usual React workflow.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+The `OwnerClass` is an extension on top of the `OwneeClass` and thus has all of the same methods and properties as well as those listed below.  Please see the `OwneeClass` docs for further information on these properties.
+
+### <a id="Requiring-OwnerClass"></a>Requiring OwnerClass [#](#Requiring-OwnerClass)
+The `OwnerClass` is exposed as a nethod under `tux/React` via:
+
+```javascript
```

Comment:
In your example starting on line 20, you use the `React.createOwnerClass` method, whereas in this example you're assigning the createOwnerClass function to a variable. Would it be better to maintain some consistency between the examples or note why you're doing one versus another?